app/controllers/bulk_edit_controller.rb

```diff
@@ -0,0 +1,40 @@
+# frozen_string_literal: true
+class BulkEditController < ApplicationController
+  include Blacklight::SearchHelper
+
+  def resources_edit
+    (solr_response, _document_list) = search_results(q: params["q"], f: params["f"])
+    @resources_count = solr_response["response"]["numFound"]
+  end
+
+  def resources_update
+    args = {}.tap do |hash|
```

Comment:
Why not just do
```ruby
args = { mark_complete: params["mark_complete"] == "1 }
```
?

Reply:
To set the pattern for adding more arguments. lmk if you prefer to change it for now.

Reply:
If you like it the way it is I won't argue.

Reply:
Okay because I just added Another One! hahaha *tap happy*